src/client/game.cpp

```diff
@@ -3831,13 +3835,20 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats, f32 dtime,
 	*/
 	runData.update_draw_list_timer += dtime;
 
+	float update_draw_list_delta = 0.2f;
+	if (ShadowRenderer *shadow = RenderingEngine::get_shadow_renderer())
```

Comment:
arf a new singleton call... i'm working slowly to remove them :(

Reply:
sorry, I didn't find another way to use it in here.